conda_forge_tick/make_graph.py

```diff
@@ -28,6 +28,11 @@ def make_graph(names, gx=None):
 
     for i, name in enumerate(total_names):
         try:
+            sub_graph = {
```

Comment:
Should we pull the generation of the dict into its own function? That would cut down on the `if name in new_names...` boilerplate (since the function can return a dict at various points).